activiti-services/activiti-services-query/activiti-services-query-rest/src/test/java/org/activiti/services/query/rest/support/TestMvcClient.java

```diff
@@ -0,0 +1,365 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.services.query.rest.support;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+import org.springframework.hateoas.Link;
+import org.springframework.hateoas.LinkDiscoverer;
+import org.springframework.hateoas.LinkDiscoverers;
+import org.springframework.http.*;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.*;
+import org.springframework.util.Assert;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+/**
+ * Helper methods for rest api web integration testing from spring-data-rest-tests-core module
+ * 
+ */
+public class TestMvcClient {
+
+	public static MediaType DEFAULT_MEDIA_TYPE = org.springframework.hateoas.MediaTypes.HAL_JSON;
+
+	private final MockMvc mvc;
+	private final LinkDiscoverers discoverers;
+	private String basePath = "/"; 
+
+	/**
+	 * Creates a new {@link TestMvcClient} for the given {@link MockMvc} and {@link LinkDiscoverers}.
+	 * 
+	 * @param mvc must not be {@literal null}.
+	 * @param discoverers must not be {@literal null}.
+	 */
+	public TestMvcClient(MockMvc mvc, LinkDiscoverers discoverers) {
+
+		Assert.notNull(mvc, "MockMvc must not be null!");
+		Assert.notNull(discoverers, "LinkDiscoverers must not be null!");
+
+		this.mvc = mvc;
+		this.discoverers = discoverers;
+	}
+	
+	/**
+	 * Set base path uri. Default value is "/"
+	 * 
+	 * @param uri basePath 
+	 * @return TestMvcClient instance
+	 */
+	public TestMvcClient setBasePath(String uri) {
+		this.basePath = uri;
+		
+		return this;
+	}
+
+	/**
+	 * Initializes web tests. Will register a {@link MockHttpServletRequest} for the current thread.
+	 */
+	public static void initWebTest() {
+
+		MockHttpServletRequest request = new MockHttpServletRequest();
+		ServletRequestAttributes requestAttributes = new ServletRequestAttributes(request);
+		RequestContextHolder.setRequestAttributes(requestAttributes);
+	}
+
+	public static void assertAllowHeaders(HttpEntity<?> response, HttpMethod... methods) {
+
+		HttpHeaders headers = response.getHeaders();
+
+		assertThat(headers.getAllow(), hasSize(methods.length));
+		assertThat(headers.getAllow(), hasItems(methods));

```

Comment:
Could you use AssertJ, please?
`assertThat(headers.getAllow()).containsExactly(methods)`
This will already check the size.